packages/astro/test/i18n-routing.test.js

```diff
@@ -219,6 +219,14 @@ describe('[DEV] i18n routing', () => {
 			expect(await response2.text()).includes('Hello world');
 		});
 
+		it('should only match the default locale string in URLs where it is actually used as a locale', async () => {
+			// It should _not_ match `/en` from `/spanish/enigma` 
+			// and should respond with a 200
+			const response = await fixture.fetch('/new-site/spanish/enigma');
+			expect(response.status).to.equal(200);
+			expect(await response.text()).includes('Astro Enigma');
```

Comment:
I believe you'd want to add an assertion for "Current Locale: es", not "Astro Enigma"